ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1390,6 +1390,21 @@ vlan_interfaces:
     isis_network_point_to_point: < boolean >
     mtu: < mtu >
     no_autostate: < true | false >
+    # New improved "vrrp" data model to support multiple IDs
+    vrrp_ids:
+      - id: < vrid >
+        priority: < instance_priority >
+        advertisement_interval: < advertisement_interval>
```

Comment:
Similar as previous, advertisement has 2 different options in EOS, mismatch being the second one:
```shell
vrrp 100 advertisement mismatch action ?
  drop  Drop the mismatched advertisement packets
  log   Log the mismatched advertisement packets
```
Should we create 2 separate keywords OR if we want to expand in the future we can use "advertisement_mismatch_action"? Second one should be easier

Reply:
Just discussed with @ClausHolbechArista . Could you please update the data model to the following?

```yaml
vlan_interfaces:
  < Vlan_id_1 >:
    # New improved "vrrp" data model to support multiple IDs
    vrrp_ids:
      - id: < vrid >
        priority_level: < instance_priority >       <--Change
        advertisement:                                         <--Change
          interval: < advertisement_interval>     <--Change
        preempt:
          enabled: < true | false >
          delay:
            minimum: < minimum_preemption_delay >
            reload: < minimum_preemption_delay >
        ipv4:
          address: < virtual_ip_address >
        ipv6:
          address: < virtual_ip_address >
```